images/ubuntu/editor/Dockerfile

```diff
@@ -24,6 +24,18 @@ RUN for mod in $module; do \
 	# Set execute permissions for modules
 	&& chmod -R 755 /opt/unity/editors/$version/Editor/Data/PlaybackEngines
 
+RUN echo "$version-$module" | grep -q -v '^2021.[2-3].*linux' \
```

Comment:
Note that in `unity-builder` linux non-il2cpp builds are built using `base`, not `linux` [currently](https://github.com/game-ci/unity-builder/blob/d975f3b80f2c02df508066ce2a8ac0bc723d4507/src/model/image-tag.ts#L40).

Reply:
Do we have an example workflow that bootstraps a server build? Does the public api of builder have to change or does it work already?

@davidmfinol do you know?

Reply:
So we need a build with the linux-mono module?

Reply:
Check.